src/pages/ProfilesExplorerView/components/SceneExploreDiffFlameGraph/components/SceneDiffFlameGraph/SceneDiffFlameGraph.tsx

```diff
@@ -115,7 +118,23 @@ export class SceneDiffFlameGraph extends SceneObjectBase<SceneDiffFlameGraphStat
     };
   };
 
-  static Component = ({ model }: SceneComponentProps<SceneDiffFlameGraph>) => {
+  onClickAutoSelect = () => {
+    reportInteraction('g_pyroscope_app_diff_auto_select_clicked');
+
+    this.publishEvent(new EventDiffAutoSelect({}), true);
+  };
+
+  onClickChoosePreset = () => {
+    reportInteraction('g_pyroscope_app_diff_choose_preset_clicked');
+
+    this.publishEvent(new EventDiffChoosePreset({}), true);
+  };
+
+  onClickLearn = () => {
```

Comment:
**todo**: It's not being used and the event is not tracked at the moment.

Reply:
Ooops, my bad! It should be tracked.